Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 5
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	all
	99	discretize_counts
	100

[Fri Jan 29 23:23:51 2021]
rule discretize_counts:
    input: read_counts/AML-06-001_AD.csv, read_counts/AML-06-001_DP.csv
    output: input/AML-06-001_input.csv, input/AML-06-001.loci
    jobid: 5
    wildcards: patient=AML-06-001

[Fri Jan 29 23:23:51 2021]
rule discretize_counts:
    input: read_counts/AML-76-001_AD.csv, read_counts/AML-76-001_DP.csv
    output: input/AML-76-001_input.csv, input/AML-76-001.loci
    jobid: 76
    wildcards: patient=AML-76-001

[Fri Jan 29 23:23:51 2021]
rule discretize_counts:
    input: read_counts/AML-28-001_AD.csv, read_counts/AML-28-001_DP.csv
    output: input/AML-28-001_input.csv, input/AML-28-001.loci
    jobid: 30
    wildcards: patient=AML-28-001


[Fri Jan 29 23:23:51 2021]
rule discretize_counts:
    input: read_counts/AML-26-001_AD.csv, read_counts/AML-26-001_DP.csv
    output: input/AML-26-001_input.csv, input/AML-26-001.loci
    jobid: 28
    wildcards: patient=AML-26-001


[Fri Jan 29 23:23:51 2021]
rule discretize_counts:
    input: read_counts/AML-01-001_AD.csv, read_counts/AML-01-001_DP.csv
    output: input/AML-01-001_input.csv, input/AML-01-001.loci
    jobid: 1
    wildcards: patient=AML-01-001

[Fri Jan 29 23:23:56 2021]
Error in rule discretize_counts:
    jobid: 1
    output: input/AML-01-001_input.csv, input/AML-01-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpbum9lhhm.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:56 2021]
rule discretize_counts:
    input: read_counts/AML-31-001_AD.csv, read_counts/AML-31-001_DP.csv
    output: input/AML-31-001_input.csv, input/AML-31-001.loci
    jobid: 33
    wildcards: patient=AML-31-001

[Fri Jan 29 23:23:56 2021]
Error in rule discretize_counts:
    jobid: 30
    output: input/AML-28-001_input.csv, input/AML-28-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpowldx9lo.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:56 2021]
rule discretize_counts:
    input: read_counts/AML-66-001_AD.csv, read_counts/AML-66-001_DP.csv
    output: input/AML-66-001_input.csv, input/AML-66-001.loci
    jobid: 66
    wildcards: patient=AML-66-001

[Fri Jan 29 23:23:56 2021]
Error in rule discretize_counts:
    jobid: 28
    output: input/AML-26-001_input.csv, input/AML-26-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpaoebr_bu.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:56 2021]
rule discretize_counts:
    input: read_counts/AML-77-001_AD.csv, read_counts/AML-77-001_DP.csv
    output: input/AML-77-001_input.csv, input/AML-77-001.loci
    jobid: 77
    wildcards: patient=AML-77-001

[Fri Jan 29 23:23:57 2021]
Error in rule discretize_counts:
    jobid: 5
    output: input/AML-06-001_input.csv, input/AML-06-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmptfhdfh7p.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:57 2021]
rule discretize_counts:
    input: read_counts/AML-51-001_AD.csv, read_counts/AML-51-001_DP.csv
    output: input/AML-51-001_input.csv, input/AML-51-001.loci
    jobid: 51
    wildcards: patient=AML-51-001

[Fri Jan 29 23:23:57 2021]
Error in rule discretize_counts:
    jobid: 76
    output: input/AML-76-001_input.csv, input/AML-76-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpwou0dr0y.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:57 2021]
rule discretize_counts:
    input: read_counts/AML-74-001_AD.csv, read_counts/AML-74-001_DP.csv
    output: input/AML-74-001_input.csv, input/AML-74-001.loci
    jobid: 74
    wildcards: patient=AML-74-001

[Fri Jan 29 23:23:58 2021]
Error in rule discretize_counts:
    jobid: 51
    output: input/AML-51-001_input.csv, input/AML-51-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpuifq3qvk.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:58 2021]
rule discretize_counts:
    input: read_counts/AML-43-001_AD.csv, read_counts/AML-43-001_DP.csv
    output: input/AML-43-001_input.csv, input/AML-43-001.loci
    jobid: 44
    wildcards: patient=AML-43-001

[Fri Jan 29 23:24:01 2021]
Error in rule discretize_counts:
    jobid: 33
    output: input/AML-31-001_input.csv, input/AML-31-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp07fguo81.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:24:01 2021]
rule discretize_counts:
    input: read_counts/AML-14-001_AD.csv, read_counts/AML-14-001_DP.csv
    output: input/AML-14-001_input.csv, input/AML-14-001.loci
    jobid: 16
    wildcards: patient=AML-14-001

[Fri Jan 29 23:24:01 2021]
Error in rule discretize_counts:
    jobid: 77
    output: input/AML-77-001_input.csv, input/AML-77-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmplmm230uv.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:24:01 2021]
rule discretize_counts:
    input: read_counts/AML-73-001_AD.csv, read_counts/AML-73-001_DP.csv
    output: input/AML-73-001_input.csv, input/AML-73-001.loci
    jobid: 73
    wildcards: patient=AML-73-001

[Fri Jan 29 23:24:01 2021]
Error in rule discretize_counts:
    jobid: 66
    output: input/AML-66-001_input.csv, input/AML-66-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp0mqgdyil.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:24:01 2021]
rule discretize_counts:
    input: read_counts/AML-32-001_AD.csv, read_counts/AML-32-001_DP.csv
    output: input/AML-32-001_input.csv, input/AML-32-001.loci
    jobid: 34
    wildcards: patient=AML-32-001

[Fri Jan 29 23:24:02 2021]
Error in rule discretize_counts:
    jobid: 74
    output: input/AML-74-001_input.csv, input/AML-74-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpe1rrd28p.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:24:02 2021]
rule discretize_counts:
    input: read_counts/AML-90-001_AD.csv, read_counts/AML-90-001_DP.csv
    output: input/AML-90-001_input.csv, input/AML-90-001.loci
    jobid: 90
    wildcards: patient=AML-90-001

[Fri Jan 29 23:24:03 2021]
Error in rule discretize_counts:
    jobid: 44
    output: input/AML-43-001_input.csv, input/AML-43-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp0c7qbc6f.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:24:03 2021]
rule discretize_counts:
    input: read_counts/AML-41-001_AD.csv, read_counts/AML-41-001_DP.csv
    output: input/AML-41-001_input.csv, input/AML-41-001.loci
    jobid: 42
    wildcards: patient=AML-41-001

[Fri Jan 29 23:24:06 2021]
Error in rule discretize_counts:
    jobid: 16
    output: input/AML-14-001_input.csv, input/AML-14-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpdohjjppb.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:24:06 2021]
rule discretize_counts:
    input: read_counts/AML-09-001_AD.csv, read_counts/AML-09-001_DP.csv
    output: input/AML-09-001_input.csv, input/AML-09-001.loci
    jobid: 8
    wildcards: patient=AML-09-001

[Fri Jan 29 23:24:06 2021]
Error in rule discretize_counts:
    jobid: 34
    output: input/AML-32-001_input.csv, input/AML-32-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpoqkd3y8q.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:24:06 2021]
rule discretize_counts:
    input: read_counts/AML-67-001_AD.csv, read_counts/AML-67-001_DP.csv
    output: input/AML-67-001_input.csv, input/AML-67-001.loci
    jobid: 67
    wildcards: patient=AML-67-001

[Fri Jan 29 23:24:06 2021]
Error in rule discretize_counts:
    jobid: 90
    output: input/AML-90-001_input.csv, input/AML-90-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpa07jpwxw.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:24:06 2021]
rule discretize_counts:
    input: read_counts/AML-02-001_AD.csv, read_counts/AML-02-001_DP.csv
    output: input/AML-02-001_input.csv, input/AML-02-001.loci
    jobid: 2
    wildcards: patient=AML-02-001

[Fri Jan 29 23:24:06 2021]
Error in rule discretize_counts:
    jobid: 73
    output: input/AML-73-001_input.csv, input/AML-73-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp2ej_qnvg.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:24:06 2021]
rule discretize_counts:
    input: read_counts/AML-83-001_AD.csv, read_counts/AML-83-001_DP.csv
    output: input/AML-83-001_input.csv, input/AML-83-001.loci
    jobid: 83
    wildcards: patient=AML-83-001

[Fri Jan 29 23:24:07 2021]
Error in rule discretize_counts:
    jobid: 42
    output: input/AML-41-001_input.csv, input/AML-41-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpndm_ivgn.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:24:07 2021]
rule discretize_counts:
    input: read_counts/AML-13-001_AD.csv, read_counts/AML-13-001_DP.csv
    output: input/AML-13-001_input.csv, input/AML-13-001.loci
    jobid: 15
    wildcards: patient=AML-13-001

[Fri Jan 29 23:24:10 2021]
Error in rule discretize_counts:
    jobid: 83
    output: input/AML-83-001_input.csv, input/AML-83-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp5mjd_2cl.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:24:10 2021]
rule discretize_counts:
    input: read_counts/AML-40-001_AD.csv, read_counts/AML-40-001_DP.csv
    output: input/AML-40-001_input.csv, input/AML-40-001.loci
    jobid: 41
    wildcards: patient=AML-40-001

[Fri Jan 29 23:24:10 2021]
Error in rule discretize_counts:
    jobid: 8
    output: input/AML-09-001_input.csv, input/AML-09-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpnvm89q9b.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:24:10 2021]
rule discretize_counts:
    input: read_counts/AML-100-001_AD.csv, read_counts/AML-100-001_DP.csv
    output: input/AML-100-001_input.csv, input/AML-100-001.loci
    jobid: 9
    wildcards: patient=AML-100-001

Terminating processes on user request, this might take some time.
[Fri Jan 29 23:24:10 2021]
Error in rule discretize_counts:
    jobid: 9
    output: input/AML-100-001_input.csv, input/AML-100-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpyr4xiqa0.discretize_counts.R' died with <Signals.SIGINT: 2>.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
[Fri Jan 29 23:24:10 2021]
Error in rule discretize_counts:
    jobid: 41
    output: input/AML-40-001_input.csv, input/AML-40-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpwkisylde.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
[Fri Jan 29 23:24:10 2021]
[Fri Jan 29 23:24:10 2021]
Error in rule discretize_counts:
Error in rule discretize_counts:
    jobid: 2
    jobid: 67
    output: input/AML-02-001_input.csv, input/AML-02-001.loci
    output: input/AML-67-001_input.csv, input/AML-67-001.loci
[Fri Jan 29 23:24:10 2021]


Error in rule discretize_counts:
    jobid: 15
    output: input/AML-13-001_input.csv, input/AML-13-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpx0km5lk5.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpjiklcald.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpwgyt4dqn.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
Job failed, going on with independent jobs.
Complete log: /scratch/data/leah/phyolin_private/aml_patients/.snakemake/log/2021-01-29T232351.471444.snakemake.log
